src/org/javarosa/xml/util/ActionableInvalidStructureException.java

```diff
@@ -0,0 +1,37 @@
+package org.javarosa.xml.util;
+
+import org.javarosa.core.services.locale.Localization;
+import org.javarosa.core.util.NoLocalizedTextException;
+
+/**
+ * Invalid structure error that can _potentially_ be recovered from via
+ * advanced user intervention. Useful for notifying the user that the issue lies
+ * on the server.
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class ActionableInvalidStructureException extends InvalidStructureException {
```

Comment:
This class isn't used in the JavaRosa codebase at all. I guess it's used in dimagi's client but looks like we don't need it. At least yet?